code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -272,3 +272,41 @@
 	icon_state = "scarecrow_sack"
 	item_state = "scarecrow_sack"
 	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+
+/obj/item/clothing/mask/gondola
+	name = "gondola mask"
+	desc = "Genuine gondola fur."
+	icon_state = "gondola"
+	item_state = "gondola"
+	flags_inv = HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/clothing/mask/gondola/speechModification(M)
+	if(copytext(M, 1, 2) != "*")
+		M = " [M]"
+		var/list/spurdo_words = strings("spurdo_replacement.json", "spurdo")

```

Comment:
denied immediately for porting spurdo mask. goof PR in three days removing it because frog mask saying "ree" every three seconds is annoying.

Reply:
frog mask was roundstart and i think it made sounds too this doesnt, hell, let's call the big man himself @Iamgoofball thoughts?

Reply:
I think his problem was the sound, not the to_chat.

Reply:
this isnt spamming reee, this is a proper voice changer

Reply:
Goof's opinion also doesn't matter, just the maintainers :^)

Reply:
@ExcessiveUseOfCobblestone what's the differencde